katex.js

```diff
@@ -189,4 +189,12 @@ export default {
      * adds a new macro to builtin macro list
      */
     __defineMacro: defineMacro,
+    /**
+     * Exposes the dom tree, which can be useful for type checking against nodes.
```

Comment:
Maybe this should be rewritten from "dom tree" to something like "dom tree node types"?